apps/smart-contracts/core/test/unit/TokenSender.test.ts

```diff
@@ -186,15 +175,103 @@ describe('=> TokenSender', () => {
     })
   })
 
-  describe('# send', () => {
+  describe('# setAccountLimitResetPeriod', () => {
+    it('reverts if not role holder', async () => {
+      expect(
+        await tokenSender.hasRole(
+          await tokenSender.SET_ACCOUNT_LIMIT_RESET_PERIOD_ROLE(),
+          user.address
+        )
+      ).to.eq(false)
```

Comment:
```suggestion
      ).eq(false)
```
similarly for rest of file